tools/prowgen/cmd/prowgen/main.go

```diff
@@ -152,20 +161,20 @@ func main() {
 		// Store the job config generated from all meta-config files in a cache map, and combine the
 		// job configs before we generate the final config files.
 		// In this way we can have multiple meta-config files for the same org/repo:branch
-		cachedOutput := map[ref]k8sProwConfig.JobConfig{}
+		cachedOutput := map[ref]*k8sProwConfig.JobConfig{}
```

Comment:
I was thinking using pointer is more efficient since these structs are relatively large, but I agree it's more readable when directly using values. I have removed the pointers here and elsewhere.